mantidimaging/core/io/test/io_test.py

```diff
@@ -364,10 +364,7 @@ def test_save_recon_to_nexus(self):
             self.assertIn(str(datetime.date.today()),
                           _nexus_dataset_to_string(nexus_file[recon_name]["reconstruction"]["date"]))
 
-            assert abs(
-                np.max(
-                    np.array(nexus_file[recon_name]["data"]["data"]) -
-                    _rescale_recon_data(recon.data).astype("uint16"))) <= 1
+            assert abs(np.max(np.array(nexus_file[recon_name]["data"]["data"]) - recon.data)) <= 1
```

Comment:
This is checking that the data stored in the nexus is close to the original data?

`numpy.testing.assert_array_equal` or `assert_allclose`?